### ISSUE TITLE
Multiple loop profiles for sick days, vacations etc.

### ISSUE THREAD

**Issue by jenm999** [LoopKit/Loop#331]
Many people are using multiple instances of Loop on the same phone and switching between them for weekends, vacations, sick days, etc.. Since most settings are constant - Riley connection, Dex transmitter, Nightscout, DIA etc.  - I think it would be nice to have the option to create multiple profiles for the variables that change in the event of sickness or travel. Just for ISF, carb ratios and basal rates.

**Comment by ArkonPhoenix** [comment]
Yes please.

**Comment by rfmurphy81** [comment]
Can this be closed out now that overrides are in Dev?

**Comment by NightScoutForSeb** [comment]
Overrides do not cover the need for multiple profiles. Eg school days versus weekend days, days of the monthly cycle.

**Comment by MrBennington** [comment]
I really support the idea of having different profiles for your settings like basals, cr, isf. Because the process of changing and finetuning them is an ever ongoing task for most users and profiles would help to be able to revert to the last settings. Most users keep screenshots of those settings which indicates there's a need for such a feature.

Overrides work very well for special events with a limited duration. But their integration into the ux is not fully consistent especially for longer use. 

For example when have your "eating soon" enabled, your override is disabled and not reenabled automaticallya afterwards. Also they are a more rough estimated approach compared to setting your exact factors.

Profiles could be used for mid/long term changes to your basals and factors. So you can react to a potential change in sensitivity or for using a site (like my arms) that has a different absorption, and therefore the changes will persist for the next pod cycle or likewise several days for other reasons. 

These profiles could be integrated into the settings. With a "create from current settings", "load", "rename" and "delete" functionality.

**Comment by JELCRAWFORD** [comment]
Agreed with above.  Separate profiles also permits you to still use overrides. Currently if you are using overrides to manage sensitivity issues from a site change and/or for days or longer time frame, then you no longer can also use an override for individual variations or mistakes like underestimated carbs. Overrides are great for temporary problems, fine tuning day-to day differences making them  invaluable for daily management but not ideal for settings changes that switch back and forth over days.

**Comment by FreThor** [comment]
If this is implemented, please have them include ALL settings (well, maybe no need for pump/CGM), but please include max bolus/basal limits, ratios, basal program, and even the set of overrides.
